core/commands/commit_compare.py

```diff
@@ -172,6 +172,11 @@ class GsCompareAgainstCommand(PanelActionMixin, WindowCommand, GitCommand):
     def run(self, target_commit=None, file_path=None, current_file=False):
         self._file_path = self.file_path if current_file else file_path
         self._target_commit = target_commit
+
+        # if there is a target, show option to compare against HEAD
+        if target_commit:
+            self.default_actions.insert(0, ["compare_against_head", "HEAD"])
```

Comment:
This will mutate the static class element.

```python
class T:
    array = []
    def print(self):
        print(self.array)
    def add(self, num):
        self.array.insert(0, num)

a = T()
b = T()

a.print()
b.print()
b.add(1)
b.print()
a.print()

# []
# []
# [1]
# [1]
```

Instead, you could do something like:
```python
self.default_actions = [["compare_against_head", "HEAD]] + self.default_actions
```